osquery/tables/system/windows/windows_crashes.cpp

```diff
@@ -0,0 +1,674 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winternl.h>
+#pragma warning(push)
+// C:\Program Files (x86)\Windows Kits\8.1\Include\um\DbgHelp.h(3190):
+// warning C4091: 'typedef ': ignored on left of '' when no variable is
+// declared
+#pragma warning(disable : 4091)
+#include <DbgHelp.h>
+#pragma warning(pop)
+#include <DbgEng.h>
+
+#include <boost/algorithm/string/predicate.hpp>
+#include <boost/filesystem.hpp>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/process.h"
```

Comment:
Can you regroup these header includes to ensure that relative paths in quotes come after the SDK header includes in brackets?